src/program.ts

```diff
@@ -37,10 +37,7 @@ export function run<Init, State, Action, View>(
   const managerTeardowns: Array<() => void> = [];
   let isRunning = false;
   let isProcessing = false;
-  const actionQueue: Array<{
-    dispatch: Dispatch<unknown>;
-    action: unknown;
-  }> = [];
+  const actionQueue: Array<() => void> = [];
```

Comment:
In general I like storing as much pure data as possible because that makes it easier to inspect the state. If the action is stored as pure data, it is easier to inspect than if it is encapsulated in a lambda/thunk.

Reply:
Fair enough.